packages/flutter/lib/src/widgets/navigation_toolbar.dart

```diff
@@ -32,8 +32,11 @@ class NavigationToolbar extends StatelessWidget {
     this.trailing,
     this.centerMiddle = true,
     this.middleSpacing = kMiddleSpacing,
+    this.defaultToolbarHeight,
```

Comment:
Why are we calling this `defaultToolbarHeight` instead of just `toolbarHeight`?

Reply:
I think we should probably ditch this along with centerIcons for now, or move it to a separate PR.